### PR TITLE
docs(ld-badge): add react examples

### PR DESCRIPTION
**Pull request by borisdiakur**
# Description

Adds React examples to ld-badge docs.

## Type of change

- [x] Documentation content changes

## Is it a breaking change?

- [x] No

# How Has This Been Tested?

- [x] tested manually

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

File: src/liquid/components/ld-badge/readme.md [hunk 1]
```diff
@@ -36,6 +40,15 @@ permalink: components/ld-badge/
   </svg>
 </ld-badge>
 
+<!-- React component -->
+
+<LdBadge>
+  <svg {...{ slot: 'icon' }} viewBox="0 0 24 24" fill="none">
```
Comment (toastedtoast) [inline src/liquid/components/ld-badge/readme.md:46]: Why not just slot="icon"? 🤷‍♂️

Comment (borisdiakur) [reply src/liquid/components/ld-badge/readme.md:46]: > Why not just slot="icon"? 🤷‍♂️

Typescript would complain. It doesn't know slot. 😣

Comment (toastedtoast) [reply src/liquid/components/ld-badge/readme.md:46]: Damn, expected that answer. 😂
Is there any way to extend the React type? 🤔

Comment (borisdiakur) [reply src/liquid/components/ld-badge/readme.md:46]: This seems to be an issue with our sandbox app. The type is missing on the svg element. If you use a span, it's there. So it's somehow available on HTMLAttributes but missing on SVGProps. I don't think it's an issue with our library.

**Pull request closed without merging**